### DISCUSSION TITLE
Someof with no elements, comes with response [""], [] expected

### DISCUSSION THREAD

**Discussion by jpeter06** [mockoon/mockoon#1274] [Help]
Hello all,
I am using **someof** with an **array**, and using triple **{{{** 
when the result is empty  the response is [ "" ]
should be: []
`{{{someOf (array 'A'  'B' 'C') 0 2 true}}}`
Any Help?

**Comment by 255kb** [comment]
Hi, 
`someOf`'s last parameter is basically a "stringify" but indeed a bit broken. This is quite old code and I will [fix it for the next release](https://github.com/mockoon/mockoon/issues/1289) and update the documentation. 
In the meantime you will have more luck using the `stringify` helper: 
`{{{stringify (someOf (array 'A'  'B' 'C') 0 2)}}}`
